src/broker/exec.c

```diff
@@ -30,34 +30,34 @@
 
 static void exec_finalize (void *arg)
 {
-    flux_subprocess_server_t *s = arg;
-    flux_subprocess_server_stop (s);
+    subprocess_server_t *s = arg;
+    subprocess_server_stop (s);
 }
 
 void exec_terminate_subprocesses (flux_t *h)
 {
-    flux_subprocess_server_t *s = flux_aux_get (h, "flux::exec");
+    subprocess_server_t *s = flux_aux_get (h, "flux::exec");
 
     /* exec_initialize() never called */
     if (!s)
         return;
 
-    if (flux_subprocess_server_subprocesses_kill (s,
+    if (subprocess_server_subprocesses_kill (s,
                                                   SIGTERM,
                                                   EXEC_TERMINATE_TIMEOUT) < 0)
-        flux_log_error (h, "flux_subprocess_server_subprocesses_kill");
+        flux_log_error (h, "subprocess_server_subprocesses_kill");
```

Comment:
nit: in ` libsubprocess: drop flux_ prefix from server API ` correct tabbing (but appears code will disappear in later commit)

Reply:
Eh, whoops.  I think this is probably not worth the effort to refactor the commits though, given that the PR doesn't leave behind a style issue.